endstate_correction/protocol.py

```diff
@@ -8,21 +8,22 @@
 import pandas as pd
 from openmm.app import Simulation
 from pymbar import MBAR
+from openmm.unit import Quantity
 
 
 @dataclass
 class BSSProtocol:
     """This is a dataclass mimicking the BioSimSpace.Protocol."""
 
-    timestep: int  # fs
-    runtime: float  # ns
-    temperature: float  # K
-    pressure: float  # atm
+    timestep: Quantity  # fs
```

Comment:
The reason that I created this dataclass is to have an interface between the BSS.unit and the endstate_correction system, I think the best way for an interface might be that it is only using the unitless float such that the interface could be easily called by other packages.

Reply:
Could we keep this here with units but create a conversion class that inherits this and removes the units with `timestep.value_in_unit(unit.femtosecond)`? that way we are explicit about the units.

And, I think it might also be best to set timestep as `int`, as well as the `runtime`.

I have updated this part of the code to be more explicit about what we are providing here --- I changed runtime to `n_integration_steps` steps (typehint: `int`), since that way I think we can ensure that we all understand what we are talking about. the total runtime is then `n_integration_steps` * `timestep`.